<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_window</class>
 <widget class="QDialog" name="dialog_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	font: auto &quot;Arial&quot;;
	font-size: 14px;
font-weight:bold;

}

QLabel{


}


#dialog_window{
	background-color: #ffffff;
	background-color: rgb(6, 76, 130);
}



QPushButton{
	border: none;
	min-height: 40px;
	border-radius: 5px;

	
}
/*
QPushButton:hover{

	background-color:rgb(22, 38, 76);

}

QPushButton:disabled{
	color: #808080;
	background-color:rgb(210, 210, 210);

}
*/

/*********************************************/
QLabel{
	color: #ffffff;
}


/*********************************************/


/*******************************************/
QLineEdit
{
	border:2px solid rgb(6, 76, 130);
    border-radius: 3px;
    padding: 1px 18px 1px 8px;
    min-width: 6em;
	min-height: 35px;
	font-size: 16px;
}



/*******************************************/


QGroupBox{
	font-weight: bold;
}

QLineEdit:hover{
	border: 1px solid black;
	background: #e4f0fa;
}

QLineEdit:focus {
	color: rgb(6, 76, 130);
	background: #ffffff;
	selection-background-color: rgb(255, 204, 75);
}

QSpinBox:hover{
	background: #e4f0fa;
}


/*********************************************/
QComboBox
{
	border:2px solid rgb(6, 76, 130);
    border-radius: 3px;
    padding: 1px 18px 1px 8px;
    min-width: 6em;
	min-height: 35px;
	font-size: 14px;
}

QComboBox:enabled{
color: rgb(50, 50, 50);
}

QComboBox:disabled
{
	border:2px solid rgb(210, 210, 210);
	color:(210, 210, 210);

}

QComboBox::down-arrow
{   
	image: url(:/assets/icons/icons8-arrow-48.png);
	width: 15px;
    height: 15px;

}

QComboBox::drop-down
{
	 background-color:rgb(217, 82, 48);
	 min-width: 30px;
}

QComboBox::drop-down:disabled 
{
	 background-color: rgb(210, 210, 210);
	 min-width: 30px;
}

QComboBox QAbstractItemView {
    border: none;
    selection-background-color: rgb(6, 76, 130);
	selection-color: rgb(6, 76, 130);


}

QComboBox QAbstractItemView {
    border: none;
}

QComboBox QAbstractItemView::item {
    border: none;
	height:30px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>25</number>
   </property>
   <property name="leftMargin">
    <number>60</number>
   </property>
   <property name="topMargin">
    <number>80</number>
   </property>
   <property name="rightMargin">
    <number>60</number>
   </property>
   <property name="bottomMargin">
    <number>80</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>50</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="sample_name_input">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="standards_name_combobox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="error_lbl">
     <property name="maximumSize">
      <size>
       <width>16787</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 16px;
font-weight: bold;
color: rgb(255, 95, 84);
padding:5px;

min-width: 300px;
max-width: 16777px;
</string>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="horizontalFrame">
     <property name="styleSheet">
      <string notr="true">QPushButton{

background-color: #ffffff;
color: rgb(6, 76, 130);
max-width: 200px;
}


</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="run_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{

background-color: rgb(58, 209, 154);
color:#ffffff;
}</string>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(222, 222, 222);
}

QPushButton:hover{
background-color:rgb(255, 95, 84);
color:#ffffff;
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Assets/Assets.qrc"/>
 </resources>
 <connections/>
</ui>
